packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -19,10 +19,11 @@ h| Control
 h| Description
 
 | 1.1 Vulnerability reports
-| * Publish the method and/or point of contact for security reports on your website
-  ** Example: Using the https://securitytxt.org[security.txt] standard
-* Develop and document procedures for triaging reported vulnerabilities
+| * Publish a vulnerability disclosure policy on your website, defining the method and/or point of contact for security reports. The policy should list the scope for testing and include legal safe harbor authorizing testing
+  ** Example: Using template language from https://disclose.io[disclose.io] for a vulnerability disclosure policy, and the https://securitytxt.org[security.txt] standard to refer researchers to your policy
```

Comment:
```suggestion
  ** Example: Generate a policy with https://disclose.io[disclose.io] and publish it using https://securitytxt.org[RFC 9116].
```

Reply:
Since an organization would likely host a VDP on a separate page on their website and then include a link to it via security.txt, maybe the following language?

`** Example: Publish a policy generated with https://disclose.io[disclose.io] and refer to it using https://securitytxt.org[security.txt (RFC 9116)].`

Reply:
I think the new suggestion makes sense here (recreated below for ease of applying).

```suggestion
  ** Example: Publish a policy generated with https://disclose.io[disclose.io] and refer to it using https://securitytxt.org[security.txt (RFC 9116)].
```

Note: We should ensure that the controls FAQ is updated here to cover definitions of `safe harbor`, and `disclosure policy` - separate PR